lib/modules/apostrophe-pieces/lib/api.js

```diff
@@ -4,11 +4,9 @@ var Promise = require('bluebird');
 
 module.exports = function(self, options) {
 
-  self.trashPrefixFields = [ 'slug' ];
-
-  self.addTrashPrefixFields = function(fields) {
-    self.trashPrefixFields = self.trashPrefixFields.concat(fields);
-  };
+  // we need to alias this so that
```

Comment:
I see how this worked, but as discussed this method will exist after all and live in apostrophe-doc-type-manager (factor it up to parent class).